vllm/model_executor/models/llama.py

```diff
@@ -373,7 +373,15 @@ def get_quant_config(self, vllm_config: VllmConfig) -> QuantizationConfig | None
         return vllm_config.quant_config
 
 
-@support_torch_compile
+def llama_model_invariants(
+    input_ids, positions, intermediate_tensors=None, inputs_embeds=None
+):
+    """Shape invariants for Llama model compilation"""
```

Comment:
Mention these get compiled away? Also can you create a tracking issue to add more of these to other models?

Reply:
FYI mm those are translated to runtime assertion for unbacked and are compiled away for backed i will add that.